static/js/script.dart

```diff
@@ -316,6 +317,18 @@ void _setEventForCheckboxChanges() {
   });
 }
 
+void _guardReportIssue() {
+  for (AnchorElement bugLink in document.querySelectorAll('a.github_issue')) {
+    bugLink.onClick.listen((event) {
+      if (!window.confirm('This link is for reporting issues for the pub site. '
+          'If you would like to report a problem with a package, please visit '
+          'its homepage or contact its developers.')) {
+        event.preventDefault();
```

Comment:
Not sure if the browser popup is the right UI, maybe a html window of our own making is better. Browser popup dialogues are something of an happening and not the nicest UI.

If we use a custom window, and we're on a package page, we could link people to the package's homepage. I'd love if we did #1522 and could take advantage of that.

The message looks fine to me. If we go with this approach, we could have a tech writer or someone review that the message is clearly understood by people in a hurry that don't understand English as well as we do. I suspect your proposed UX doesn't quite catch the kind of people that don't pay attention to which bug tracker they are on.

Reply:
> If we use a custom window, and we're on a package page, we could link people to the package's homepage. I'd love if we did #1522 and could take advantage of that.

Agreed. Shall we land this until that happens, or shall we leave it as it is now?

Reply:
I'm OK if this lands as it is, but consider if you'll want to invest some time in improving the UX a bit, or whether we'll file an issue about doing that.

Reply:
Tracking in https://github.com/dart-lang/pub-dartlang-dart/issues/1594